.github/workflows/finish-release.yml

```diff
@@ -128,10 +128,13 @@ jobs:
 
             { "type": "section", "text": { "type": "mrkdwn", "text":
             ":${{ fromJSON('["desert", "white_check_mark"]')[github.event.inputs.dry-run != 'true'] }}:
-            *<${{ steps.release.outputs.url }} | ${{ inputs.version }}> of <${{github.server_url}}/${{github.repository}}|${{github.repository}}> has been published on GitHub.*" }},
+            *Release <${{ steps.release.outputs.url }} | ${{ inputs.version }}> of <${{github.server_url}}/${{github.repository}}|${{github.repository}}> has been published on GitHub.*" }},
 
             { "type": "section", "text": { "type": "mrkdwn", "text":
-            ":arrow_right: Look for a CI generated PR created in <https://github.com/stackrox/release-artifacts/pulls|stackrox/release-artifacts> repository
-            and confirm that members of the @release-publishers Slack group got notified about
```

Comment:
@0x656b694d is there any notification to or review from these people necessary for the CI generated PR?

Reply:
As this is a Slack notification which mention the Slack group, I think that's all enough.